src/store/preferences.ts

```diff
@@ -6,15 +6,19 @@ import { defaultSettings, Settings } from '@/config/settings';
 
 interface PreferencesState {
   initialPreferences: Settings,
-  preferences: Settings
+  preferences: Settings,
+  wslIntegrations: Record<string, boolean | string>,
```

Comment:
Note to self: we need this because WSL integrations are stored as `Record<string, boolean>`, but at run time the value may be a string describing an error.
(There is no need to change things here.)